classes/output/sort_voting_results.php

```diff
@@ -48,27 +48,13 @@ public function __construct(\stdClass $sortvoting) {
     public function export_for_template(renderer_base $output): array {
         global $DB;
 
-        $sql = "SELECT so.id,
-                    ROUND(AVG(sa.position), 2) AS avg,
-                    so.text
-                FROM {sortvoting_answers} sa
-                    JOIN {sortvoting_options} so
-                        ON sa.optionid = so.id
-                WHERE so.sortvotingid = :sortvotingid
-                GROUP BY so.id
-                ORDER BY avg ASC";
-        $existingvotes = $DB->get_records_sql($sql, ['sortvotingid' => $this->sortvoting->id]);
+        $cm = get_coursemodule_from_instance('sortvoting', $this->sortvoting->id, $this->sortvoting->course);
+        $sortvoting = $DB->get_record('sortvoting', ['id' => $this->sortvoting->id], '*', MUST_EXIST);
```

Comment:
Don't you already have this in `$this->sortvoting`? 

Retrieved here: https://github.com/odeialba/moodle-mod_sortvoting/blob/708a9a0388c2e25bcbf4a40a869c3e321314a0a4/report.php#L33

& passed to this class here: https://github.com/odeialba/moodle-mod_sortvoting/blob/708a9a0388c2e25bcbf4a40a869c3e321314a0a4/report.php#L58